src/Commands/MinifyApplicationCommand.php

```diff
@@ -61,4 +62,50 @@ protected function cleanUpEnvFile(string $appPath): void
 
         file_put_contents($envFile, $envValues);
     }
+
+    protected function removeIgnoredFilesAndFolders(string $appPath): void
+    {
+        $this->info('Cleaning up ignored files and folders…');
+
+        $itemsToRemove = config('nativephp.cleanup_exclude_files', []);
+
+        foreach ($itemsToRemove as $item) {
+            $fullPath = $appPath.'/'.$item;
+
+            if (file_exists($fullPath)) {
+                if (is_dir($fullPath)) {
+                    $this->delete_directory_recursive($fullPath);
+                } else {
+                    array_map('unlink', glob($fullPath));
+                }
+            } else {
+                foreach (glob($item) as $pathFound) {
+                    unlink($pathFound);
+                }
+            }
+        }
+    }
+
+    private function delete_directory_recursive($dir)
+    {
+        if (! file_exists($dir)) {
```

Comment:
space

Reply:
I'm not sure I'm clear on what you mean by "space".  Can you please elaborate?

Reply:
So sorry, I mean remove space after `!`
```php
if (!file_exists($dir)) {...}
```
Because, code style is main goal in clear code

Reply:
For reference, it's being applied via the `not_operator_with_successor_space` PHP-CS-Fixer rule

Reply:
So sorry! Thank you for your explain, It is my mistake!